src/tester/test.ts

```diff
@@ -22,6 +22,15 @@ async function main(options: TesterOptions, nProcesses: number, all: boolean): P
     const log = loggerWithErrors()[0];
     const dt = await getDefinitelyTyped(options, log);
     await parseDefinitions(dt, { nProcesses, definitelyTypedPath: options.definitelyTypedPath }, log);
-    await checkParseResults(/*includeNpmChecks*/false, dt, options, new UncachedNpmInfoClient());
+    try {
+        await checkParseResults(/*includeNpmChecks*/false, dt, options, new UncachedNpmInfoClient());
+    }
+    catch (e) {
+        if (!all) {

```

Comment:
this is the actual change. I decided to skip printing for "all" because presumably the list of dependent packages will be every package that depends on another.